lib/util.js

```diff
@@ -9,7 +9,7 @@
  *
  * See the License for the specific language governing permissions and limitations under the License.
  */
-
+/* globals Uint8Array */
```

Comment:
Interesting - I wonder if it's better instead to a) not use es3 in our .eslintrc file or b) if possible set es5 for this method only